theme-utils.mjs

```diff
@@ -674,7 +674,11 @@ async function versionBumpThemes() {
 	//version bump the root project if there were changes to any of the themes
 	const rootHasVersionBump = await checkProjectForVersionBump(hash);
 
-	if (versionBumpCount > 0 && !rootHasVersionBump) {
+	if (!rootHasVersionBump) {
+		console.log('No changes detected; version bump not required');
+	}
+
+	if (versionBumpCount > 0 && rootHasVersionBump) {
```

Comment:
I have had to update this logic because otherwise the version wasn't bumped when changed were made. See https://github.com/Automattic/themes/commit/2ba4dd2aa9e29d6ad566564befb974f18479c674#diff-877b4284538ebb4cf1411cf4487c456bda52082b9e72e4a0b6770d5a80d7d5a9R677

`versionBumpCount` is an count for the theme bumps done for changed themes.

`rootHasVersionBump` is a boolean that tell us if the root package version has changed from the previews release tag.

Therefore, we can skip the root package bump only when `versionBumpCount === 0` and we must bump it when it's not done yet `rootHasVersionBump === false`.

cc: @dsas

Reply:
Thanks @miksansegundo 👍